app/styles/main.scss

```diff
@@ -23,6 +23,17 @@ html, body {
   height: 100%;
 }
 
+.flexbox-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 100vh;
+}
+
+.flexbox-spacer {
+  flex-grow: 2;
```

Comment:
I still think you need to give the centered layout more time though. I totally understand how weird it looks at first glance, and it must be way weirder for you since you've seen it the other way for much longer than I have. I really think the body is king though, and the user knows where to find the nav if they need it.